pkg/aspect/root/flags/interceptor.go

```diff
@@ -86,32 +86,68 @@ func FlagsInterceptor(streams ioutils.Streams) interceptors.Interceptor {
 		if cfgFile != "" {
 			// Use config file from the flag.
 			viper.SetConfigFile(cfgFile)
+			viper.AutomaticEnv()
+			if err := viper.ReadInConfig(); err == nil {
```

Comment:
What if reading the config fails?

Reply:
Similar to the empty code blocks comment below, I wasn't sure how we wanted to deal with errors. Also why this pr is a draft as I don't want it merged without error handling